exercises/practice/pig-latin/.meta/example.cpp

```diff
@@ -0,0 +1,31 @@
+#include "pig_latin.h"
+
+namespace pig_latin {
+std::string translate_word(std::string word) {
+    for (std::string start : {"a", "e", "i", "o", "u", "yt", "xr"}) {
+        if (word.substr(0, start.size()) == start) return word + "ay";
+    }
+    for (std::string start : {"thr", "sch", "squ", "ch", "qu", "th", "rh"}) {
```

Comment:
While this list of consonant clusters is sufficient to pass the tests, those cover only a fraction of the possibilities. Words for which the current implementation would not give the correct pig latin version include "friend", "splash", "smile", "string".

That's why I said in some other comment that my implementation is more generic. Instead of explicitly listing consonant clusters, it moves characters to the back until it finds a vowel. Only `qu` needs special handling in that case.

Reply:
I would vote to move that discussion and possible solutions to an article for the exercise.